frontend/assets/javascripts/src/modules/form/payment/processing.js

```diff
@@ -1,119 +0,0 @@
-/*global Stripe */
-define([
-    'ajax',
-    'src/utils/helper',
-    'src/modules/form/helper/formUtil',
-    'src/modules/form/helper/serializer',
-    'src/modules/form/helper/loader',
-    'config/paymentErrorMessages',
```

Comment:
It seems like this processing file is the only place [this error message module](https://github.com/guardian/membership-frontend/blob/791eb801143f174764dc6eccabfe3523f2e28d38/frontend/assets/javascripts/config/paymentErrorMessages.js) is imported, maybe we can throw that out too?

Reply:
It looks like the PaymentGatewayErrors have been duplicated here: https://github.com/guardian/membership-frontend/blob/8e6779579a2e5758c7ef103c77dd2b200a9bd18f/frontend/assets/javascripts/src/modules/form/payment/paymentError.es6

Reply:
And presumably the errors in the 'card_error' block are now unnecessary, as Stripe Checkout handles this first stage of card validation.

Reply:
So I agree that we can probably remove that dead code too.